dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java

```diff
@@ -596,4 +652,73 @@ public static void saveMultiTree(MultiTree o, long languageId) {
 			throw new DotRuntimeException(e.toString());
 		}
 	}	
+	
+    /**
+     * Update a HTML Page Version Info Timestamp given a HTMLPage Identifier.
+     * If a MultiTree Object has been added or deleted from this page,
+     * its version_ts value needs to be updated so it can be included
+     * in future Push Publishing tasks
+     * 
+     * @param id The HTMLPage Identifier to pass in 
+     * @throws DotContentletStateException
+     * @throws DotDataException 
+     * @throws DotSecurityException 
+     *            
+     */
+    private static void updateVersionTs(String id) throws DotStateException, DotDataException, DotSecurityException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.dotmarketing.business.DotStateException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)